src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php

```diff
@@ -35,7 +37,7 @@ public function getRecipientId(): ?string
      */
     public function diffusionName(string $diffusionName): static
```

Comment:
```suggestion
    public function diffusionname(string $diffusionName): static
```
as it is not released yet?

Reply:
Wouldn't fit our CS, we use camelcase